src/fsharp/DependencyManager.Integration.fs

```diff
@@ -91,9 +98,18 @@ type ReflectionDependencyManagerProvider(theType: Type, nameProperty: PropertyIn
     interface IDependencyManagerProvider with
         member __.Name     = instance |> nameProperty
         member __.Key      = instance |> keyProperty
-        member __.ResolveDependencies(scriptDir, mainScriptName, scriptName, packageManagerTextLines, tfm) =
+        member this.ResolveDependencies(scriptDir, mainScriptName, scriptName, packageManagerTextLines, tfm) =
+            let key = (this :> IDependencyManagerProvider).Key
+            let eventValue = String.Join("\n", packageManagerTextLines)
```

Comment:
@brettfo,
should we fire an event per package (I.e. per packageManagers line) or one event for all packages per submission?

and perhaps not fire an event for Adding, Added and Failed that have already been fired?

Reply:
Updated to trigger the event for each package manager line.

I also added a test that verifies we don't duplicate events.  E.g., the following code results in 2 events being fired, which means the one for `System.Collections.Immutable` wasn't repeated for the second submission:

``` F#
script.Eval("#r \"nuget:System.Collections.Immutable\"")
script.Eval("#r \"nuget:Newtonsoft.Json\"")
```